cmd/ceremony/cert.go

```diff
@@ -142,16 +144,6 @@ var stringToKeyUsage = map[string]x509.KeyUsage{
 	"Cert Sign":         x509.KeyUsageCertSign,
 }
 
-type policyQualifier struct {
-	Id    asn1.ObjectIdentifier
-	Value string `asn1:"tag:optional,ia5"`
-}
-
-type policyInformation struct {
-	Policy     asn1.ObjectIdentifier
-	Qualifiers []policyQualifier `asn1:"tag:optional,omitempty"`
-}
-
 var (
 	oidExtensionCertificatePolicies = asn1.ObjectIdentifier{2, 5, 29, 32}
 	oidCPSQualifier                 = asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 2, 1}
```

Comment:
Looks like this is still being used somewhere, or the compiler would complain about it. Unify that usage with the new `policyasn1.CPSQualiferOID`?

Reply:
Whoops, it's unused. The compiler only complains about unused variables inside functions.